JASP-Desktop/data/datasetpackage.cpp

```diff
@@ -271,7 +279,7 @@ QVariant DataSetPackage::data(const QModelIndex &index, int role) const
 		int parColCount = columnCount(index.parent()),
 			parRowCount = rowCount(index.parent());
 
-		if(!_dataSet || index.column() >= parColCount || index.row() >= parRowCount)
+		if(!_dataSet || index.row() >= parRowCount)
```

Comment:
You assume here that the index always has a parent. Is it really so?

Reply:
Yes by definition, that is how DataSetPackage has been setup, see the `index() override`. The parent is used to discern whether or not we are in the `root, filter, label or data` subtree. When we are in the `label` subtree there is always a parent, because otherwise we couldn't be in the `label` subtree.